### ISSUE TITLE
How to Display sub items if the Menu first level is Category selected?

### ISSUE THREAD

**Issue by hanyseyedy** [wp-bootstrap/wp-bootstrap-navwalker#533]
#

#### Steps to reproduce the issue:
this is not issue or bug but it's a question. 
I have a problem in display nav walker. In WordPress Dashboard Menu When the first item selected is a category, it is not showing sub items. 
In normal mode, in depth=0 if a menu item has a sub-menu, the mega Menu will be displayed. Now I want if the first menu item was (active) and the menu item was category, however the Mega menu will still be displayed.

In other words, if I not add any sub menu item in WordPress menu setting, however the Mega menu will still be displayed in frontend. In my source if depth=0 has sub menu item mega Menu will be displayed.
#### What I expected:
I want in depth=0 if first item doesn't have any sub menu and selected item is a category, however the Mega menu will still be displayed. 
Now My question is: where in nav walker source can I handle this case?
